src/redux/books/booksSlice.js

```diff
@@ -1,13 +1,43 @@
 import { createSlice } from "@reduxjs/toolkit";
 
+const initialState = [
+  {
+    "item_id": "item1",
+    "title": "The Great Gatsby",
+    "author": "John Smith",
+    "category": "Fiction"
+  },
+  {
+    "item_id": "item2",
+    "title": "Anna Karenina",
+    "author": "Leo Tolstoy",
+    "category": "Fiction"
+  },
+  {
+    "item_id": "item3",
+    "title": "The Selfish Gene",
+    "author": "Richard Dawkins",
+    "category": "Nonfiction"
+  }
+]
+
+
 const booksSlice = createSlice({
   name: 'books',
-  initialState: [],
+  initialState,
   reducers: {
     addBook: (state, action) => {
-      state.push(action.payload);
+      const book = {
+        "item_id": `item${state.length+1}`,
```

Comment:
Good job by creating the bookSlice 👍🏼 , but using state length to generate ids is not a good idea as after removing an item then adding a new one you will end up with two items with the same id , to fix this use functions that generates random values instead exp: js uuid generator

Reply:
> Correction

The `nanoid` dependency was installed and used to assign unique identifiers to the books.